main.lua

```diff
@@ -185,45 +193,100 @@ local bg = love.graphics.newImage(art.."/bg.png")
 function love.draw()
   love.graphics.setColor(255,255,255)
 
-  local padding = 16
-
-  local lw = love.graphics.getWidth()/2 - padding*2
-  local lh = love.graphics.getHeight()/2 - padding*2
-
-  for i,player in pairs(players) do
-
-    local player_x = (i-1)%2
-    local player_y = math.floor((i-1)/2)
-
-    local lx = player_x*love.graphics.getWidth()/2 + padding
-    local ly = player_y*love.graphics.getHeight()/2 + padding
-
-    -- Draw a fun paralaxing background!
-    love.graphics.setScissor(lx,ly,lw,lh)
-    love.graphics.draw(bg,
-      lx-(player.entity:getAngle()/(math.pi/2)*lw)%lw,
-      ly,0,lw/bg:getWidth(),lh/bg:getHeight())
-    love.graphics.draw(bg,
-      lx+lw-(player.entity:getAngle()/(math.pi/2)*lw)%lw,
-      ly,0,lw/bg:getWidth(),lh/bg:getHeight())
-    love.graphics.setScissor()
-
-    -- Draw the level in relation to the current player
-    level:setPlayer(player.entity)
-
-    local draw_scale = (love.graphics.getHeight()/2)/128
-    level:draw(lx,ly,lw,lh,draw_scale)
-    --love.graphics.rectangle("line",lx,ly,lw,lh)
-    love.graphics.print("Player "..i,lx+padding,ly+padding)
```

Comment:
Why are we removing the padding? I wanted padding in to ensure that the setScissor is being used correctly.

Reply:
I thought the padding itself was a little ugly, especially the big black bars in the middle of the screen. As a compromise, I've reduced the padding from 16 to 8  and applied it to the Player Views too. It looks a lot better now the padding thickness in the middle is 16, rather than 32.